### PR TITLE
Added Kobayashi-san Chi no Maid Dragon

### PR DESCRIPTION
**Pull request by lordjbs**
- [x] 🔍 I have searched the `data.js` file and confirmed I am not adding a duplicate entry. Note: Different season of the same anime are okay to add such as Sword art online (2013) and Sword art online (2014) .
- [x] 🌻 I have added a anime year under `animeYear`. Note: Do not add ranges such as 2017-2019.
- [x] 🔗 I have added the myanimelist page under `animeLink`.
- [x] 3️⃣ I have at least three emojis listed under `animeEmoji`.
- [x] 5️⃣ I have a maximum of five emojis listed under `animeEmoji`.
- [x] 👍 My pull request has a descriptive title (such as `Added Sword Art Online` or `Added Sword Art Online: Alicization`).
- [x] 🖍️ I have placed the new show(s) or movie(s) in alphabetical order based on title. If the show or movie starts with 'the', then use the second word to alphabetize.

### CODE REVIEW

File: src/store/listAnime.js [hunk 1]
```diff
@@ -53,6 +53,12 @@ export default [
         animeYear:1998,
         animeLink:"https://myanimelist.net/anime/185/Initial_D_First_Stage"
     },
+    {
+        animeEmoji:"👩🐉👩‍💼",
+        animeName:"Kobayashi-san Chi no Maid Dragon",
+        animeYear:2017,
+        animeLink:"https://myanimelist.net/anime/33206/Kobayashi-san_Chi_no_Maid_Dragon?q=Dragon%20Maid"
```
Comment (2pai) [inline src/store/listAnime.js]: i think we can remove the query string , 
that would be nice

Comment (lordjbs) [reply src/store/listAnime.js]: sure thing

Comment (2pai) [reply src/store/listAnime.js]: please send pr again @lordjbs

Comment (lordjbs) [reply src/store/listAnime.js]: So uh, new or?

Comment (lordjbs) [reply src/store/listAnime.js]: pr*

Comment (2pai) [reply src/store/listAnime.js]: just commit again to this pr

Comment (lordjbs) [reply src/store/listAnime.js]: Done

**Pull request merged**